plugins/in_syslog/syslog.c

```diff
@@ -244,7 +244,11 @@ static struct flb_config_map config_map[] = {
      0, FLB_TRUE, offsetof(struct flb_syslog, source_address_key),
      "Key where the source address will be injected"
     },
-
+    {
+     FLB_CONFIG_MAP_STR, "frame", (char *) NULL,
```

Comment:
what about using a `format` property name ? (frame seems to be low level for the end user)